### ISSUE TITLE
Add support for Angular 17

### ISSUE THREAD

**Issue by aeslinger0** [leNicDev/ng-hcaptcha#78]
Please add a version that works with Angular 17. Currently, I am getting this error:

```
npm ERR! Found: @angular/common@17.0.7
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^17.0.7" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0" from ng-hcaptcha@2.2.0
npm ERR! node_modules/ng-hcaptcha
npm ERR!   ng-hcaptcha@"^2.2.0" from the root project
```

**Comment by ankit18singh** [comment]
+1

**Comment by ankit18singh** [comment]
+1

**Comment by reyvaxreecded** [comment]
+1

**Comment by ZenulAbidin** [comment]
While we wait for the dev to merge this commit, you can use ng-hcaptcha-transition on NPM which I set up as a temporary workaround.

**Comment by leNicDev** [comment]
Sorry for the waiting time! @max3nn's PR has been merged and v2.3.0 has been published 👍🏼
